rfcs/0044-ckb-light-client/0044-ckb-light-client.md

```diff
@@ -532,6 +532,8 @@ The parameters[\[2\]] to activate this feature are:
 | `threshold` | 75% | TBD |
 | `min_activation_epoch` | 5676 (approx. 2022-12-24 15:00:00 UTC) | TBD |
 
+For CKB Testnet, the deployment did activate at epoch 5711.
```

Comment:
Should we remove all approximate date time in the above table, or replace them with the real values?

Reply:
There is no ambiguity here, as a historical record, I think it's OK, and we will update the mainnet parameters later